src/main/java/seedu/address/ui/SuggestionTemplates.java

```diff
@@ -0,0 +1,315 @@
+        ADD_PARTICIPANT_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        ADD_MENTOR_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        ADD_TEAM_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+
+        EDIT_PARTICIPANT_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        EDIT_MENTOR_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        EDIT_TEAM_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+
+        FIND_PARTICIPANT_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        FIND_MENTOR_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        FIND_TEAM_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+
+        DELETE_PARTICIPANT_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        DELETE_MENTOR_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        DELETE_TEAM_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+
+        LIST_PARTICIPANT_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        LIST_MENTOR_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        LIST_TEAM_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+
+        SCORE_ADD_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        SCORE_SET_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        SCORE_SUB_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        LEADERBOARD_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        GET_TOP_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+
+
+        EXPORT_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        IMPORT_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+
+        HISTORY_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        UNDO_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+        REDO_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+
+        HELP_INSTRUCTION.setFill(Color.CORNFLOWERBLUE);
+
+        //Sett the template guides to Gray
+        NAME.setFill(Color.GREY);
+        PHONE.setFill(Color.GREY);
+        EMAIL.setFill(Color.GREY);
+
+        TEAM_ID.setFill(Color.GREY);
+        PARTICIPANT_ID.setFill(Color.GREY);
+        MENTOR_ID.setFill(Color.GREY);
+
+        NUMBER.setFill(Color.GREY);
+        POINTS.setFill(Color.GREY);
+        NEW_POINTS.setFill(Color.GREY);
+
+        CSV_PATH.setFill(Color.GREY);
+        PARAMETERS.setFill(Color.GREY);
+        ORGANIZATION.setFill(Color.GREY);
+        SPECIALISATION.setFill(Color.GREY);
+        PROJECT_NAME.setFill(Color.GREY);
+        SUBJECT.setFill(Color.GREY);
+        TABLE.setFill(Color.GREY);
+        ENTITY.setFill(Color.GREY);
+
+    }
+
+    /**
+     * Returns a String that represents the template form of commands.
+     * Only prefix and commands are returned.
+     * Constants that are used to guide the user(e.g NAME, PHONE_NUMBER) are not returned).
+     */
+    public static String getString(TextFlow template) {
+        String result = template.getChildren()
+                .stream()
+                .map(textElement -> (Text) textElement)
+                .map(t -> t.getText())
+                .filter(s -> !s.matches("\\[[A-Za-z{}_ ]+\\] |[A-Z_{} ]+|\\([A-Za-z_ ]+\\)"))
```

Comment:
assuming this is correct because my regex is shit

Reply:
yeap it works :)